StripeUICore/StripeUICore/Source/Elements/ElementsUI.swift

```diff
@@ -100,7 +100,7 @@ import UIKit
                                             withTextStyle: .caption1,
                                             maximumPointSize: 20)
         public var footnote = UIFont.preferredFont(forTextStyle: .footnote, weight: .regular, maximumPointSize: 20)
-        public var smallFootnote = UIFont.preferredFont(forTextStyle: .footnote, weight: .medium, maximumPointSize: 10)
+        public var smallFootnote = UIFontMetrics(forTextStyle: .footnote).scaledFont(for: .systemFont(ofSize: 10.5, weight: .medium))
```

Comment:
My slight preference is to not use half point sizes, and prefer the text styles. Can we get away with using `caption2`, it's 11.

Reply:
My assumption is that we'd want the sublabel and the error text to be the same size— is that right? If so, I'm currently sending Hannah a screenshot with the error text set to caption2, and if she says that's okay, then we can go through with it.